### DISCUSSION TITLE
Add Rocket/Mini-Rocket and MTS (Multiple Time Series) functionality for Classification, Regression, Clustering, and Anomaly Detection

### DISCUSSION THREAD

**Discussion by tsereno** [pycaret/pycaret#2511] [Ideas]
Please consider adding support for SOTA Rocket and MiniRocket for Time-Series Classification and Regression.  Please consider adding support for Time-Series clustering as well (i.e. tslearn clustering).  Having MTS (multiple time series) support would be great.

**Comment by ngupta23** [comment]
Hi @tsereno Yes, it would eventually be nice to have these in pycaret. In fact, `sktime` supports many of these and since the `pycaret` time series module is based on `sktime`, this could in theory be incorporated in pycaret as new tasks.

However, due to limited developers working on pycaret, at this time, it will not be possible to start these developments. We are looking for community users to step up and help with this so if anyone is interested in taking the lead on this development, it would be greatly appreciated. We can of course guide along the way as necessary.

**Reply by at062084** [reply]
Hi @ngupta23 Also looking for ts clustering to be included in pycaret ! Any plans to put this on the roadmap ?
